commons/commons-util/src/test/kotlin/ebi/ac/uk/util/file/ExcelReaderTest.kt

```diff
@@ -77,7 +77,29 @@ class ExcelReaderTest(private val temporaryFolder: TemporaryFolder) {
             line("file2.txt", "b1", "b2")
         }
 
-        assertThat(asTsv(testFile).readText()).isEqualTo(expectedTsv.toString().plus("\n"))
+        assertThat(asTsv(testFile)).hasContent(expectedTsv.toString().plus("\n"))
+    }
+
+    @Test
+    fun `read as TSV with linebreak`() {
+        val testFile = excel(File("${temporaryFolder.root.absolutePath}/ExcelSubmissionLineBreak.xlsx")) {
+            sheet("page tab") {
+                row {
+                    cell("Submission")
+                }
+                row {
+                    cell("Title")
+                    cell("Excel Submission \n with a line break")
+                }
+            }
+        }
+
+        val expectedTsv = tsv {
+            line("Submission")
+            line("Title", "\"Excel Submission \n with a line break\"")
+        }
+
+        assertThat(asTsv(testFile)).hasContent(expectedTsv.toString().plus("\n"))
```

Comment:
Instead of adding `plus("\n")`, I think it would be clearer to add the new line in the expected tsv

Reply:
not sure what I was thiking when did this